# HG changeset patch
# User Rik <rik@octave.org>
# Date 1358198481 28800
#      Mon Jan 14 13:21:21 2013 -0800
# Branch stable
# Node ID fe5f57f426637a26de766bf7379341455e32573b
# Parent  a653526ea4a5522485b8253dc818588de708b17d
doc: Update mkoctfile man page with newest list of configuration variables.

* doc/interpreter/mkoctfile.1: Update mkoctfile man page with newest list of
configuration variables.

diff --git a/doc/interpreter/mkoctfile.1 b/doc/interpreter/mkoctfile.1
--- a/doc/interpreter/mkoctfile.1
+++ b/doc/interpreter/mkoctfile.1
@@ -90,39 +90,39 @@ Create a MEX file.  Set the default outp
 .B \-o \fIfile\fP\fR,\fB \-\-output \fIfile\fP
 Output file name.  Default extension is \fB.oct\fP (or \fB.mex\fP if \-\-mex is
 specified) unless linking a stand-alone executable.
 .TP
 .B \-p \fIVAR\fP\fR,\fB \-\-print \fIVAR\fP
 Print configuration variable \fIVAR\fP.  Recognized variables are:
 .RS
 .Vb
-    ALL_CFLAGS                FFTW3F_LIBS
-    ALL_CXXFLAGS              FLIBS
-    ALL_FFLAGS                FPICFLAG
-    ALL_LDFLAGS               INCFLAGS
+    ALL_CFLAGS                FFTW3F_LDFLAGS
+    ALL_CXXFLAGS              FFTW3F_LIBS
+    ALL_FFLAGS                FLIBS
+    ALL_LDFLAGS               FPICFLAG
+    AR                        INCFLAGS
     BLAS_LIBS                 LAPACK_LIBS
     CC                        LDFLAGS
     CFLAGS                    LD_CXX
     CPICFLAG                  LD_STATIC_FLAG
     CPPFLAGS                  LFLAGS
-    CXX                       LIBCRUFT
-    CXXFLAGS                  LIBOCTAVE
-    CXXPICFLAG                LIBOCTINTERP
-    DEPEND_EXTRA_SED_PATTERN  LIBS
-    DEPEND_FLAGS              OCTAVE_LIBS
-    DL_LD                     OCTAVE_LINK_DEPS
-    DL_LDFLAGS                OCT_LINK_DEPS
+    CXX                       LIBOCTAVE
+    CXXFLAGS                  LIBOCTINTERP
+    CXXPICFLAG                LIBS
+    DEPEND_EXTRA_SED_PATTERN  OCTAVE_LIBS
+    DEPEND_FLAGS              OCTAVE_LINK_DEPS
+    DL_LD                     OCT_LINK_DEPS
+    DL_LDFLAGS                RANLIB
     EXEEXT                    RDYNAMIC_FLAG
     F77                       READLINE_LIBS
     F77_INTEGER_8_FLAG        SED
     FFLAGS                    XTRA_CFLAGS
     FFTW3_LDFLAGS             XTRA_CXXFLAGS
     FFTW3_LIBS
-    FFTW3F_LDFLAGS
 .Ve
 .RE
 .TP
 .B \-v\fR,\fB --verbose
 Echo commands as they are executed.
 .TP
 .B file
 Compile or link file.  Recognized file types are
